sdk/tests/conformance2/extensions/ext-texture-norm16.html

```diff
@@ -23,7 +23,7 @@
 debug("");
 
 var wtu = WebGLTestUtils;
-var gl = wtu.create3DContext();
+var gl = wtu.create3DContext(null, null, 2);
```

Comment:
Don't need this, since this is in conformance2/*.

Reply:
It's needed to support navigating directly to this test rather than running it inside the harness, which makes debugging simpler. Most of the other tests under conformance2/ do this; it was an oversight that this test didn't.